Software/app/app_template/README.md

```diff
@@ -0,0 +1,30 @@
+# app_template
+
+## Description
+
+This application provides a bloat free template for starting with application development.
+
+Developers are expected to customize this application and add their source files and build configurations according to their needs.
+
+## Configuration
+
+This application doesn't require any additional configurations to function.
```

Comment:
`doesn't` or `does not`? I'm not sure what was used until now, but please check and reflect that in this doc. I know we always use the full expression in TTS docs

Reply:
Yes you are right, we should follow TTS format, I adjusted all the doesn't in the docs for now and if we meet something else like isn't, will be updated as well.